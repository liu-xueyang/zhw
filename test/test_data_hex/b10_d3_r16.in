bcfcab90afaaa970
bce1e196dc623226
bceec9b5bb4580ba
bcfb27332fbc844b
bcfcab8c1b3b30ef
bcfcab896e34114f
bcf5c0b13c61dde1
bcfcab83f921f66d
bcf65a6c057d8e23
bcf95a773f301b6e
bcf673e5473ad5f6
bcfc62df6cae3d82
bcfcab82ca487a02
bcfaf845cf45804d
bcf82b8940e1ae04
bcf55e9db25d772c
3d2f61512ed4c7e1
3d14e812b8d08922
bcfcab92701bbcfd
bcfcab8fbef1ed57
bcfcab9311e4a36e
bcf628255b71d6c9
bcfcab8dcb99671c
3cd33f5e529a607d
bcf9aa63526604f1
bcfc483f4178fa4e
bcfcab89158e5d0d
bcf387ff45c25535
bcfcab2b1323f1d0
bcfa70f55817d254
bcf7c2c74cb181fe
bcfcab81a5ece290
bcfcab9ed0aba4f5
bcfcab9c40410eea
3d1b0622c9493add
3d330a4540fc1b04
3d36d144b2db35f1
3d22ecfec82f9123
bcf6de1244cb1909
bcfcab92535172c3
bcfcab955defa5fd
bcfcab92ccd9e573
bcf284a34378672b
bcfcab8d8e1f0e16
bced156e3cb32ae4
bcf9171d13ae29d8
bcfcab8b5d3cd9fe
bcf5d097c85963ff
3d69cee43c6e5466
3d6229c5a69a6548
bcfcaba0f9ede9a2
bcfcab9e62f7b38f
bcfcaba145d065c3
bcfcab9ec18d3197
3d1642d560d4f3ab
3d210819f4b7f014
3d33535d4616ad9b
3d29d988712a4bbd
3d01360580e87bd5
bcfcab94ce9a22f1
bcfcab97919896c0
bcfcab950d907a26
bcfcab9280b363c9
3cc470b3bc7366a3
402eaeef04cb746d
402eaed35c5f04e7
402eaeb7bdb2f4b1
402eae9c29afd73f
402eaef55860160d
402eaed9a9e88383
402eaebe04ed1cd0
402eaea26a59d8f2
402eaefbbb3f60f0
402eaee0063f070f
402eaec45a7824ac
402eaea8b8d84aec
402eaf022d8d0ac6
402eaee671855ee9
402eaecabe7610a7
402eaeaf15503427
402eaeef6c05a4d5
402eaed3c7387b0e
402eaeb82c05726a
402eae9c9b5598f6
402eaef5cddf9374
402eaeda2308065c
402eaebe8185f7af
402eaea2ea454fe6
402eaefc3ecf0e8d
402eaee08d713bd1
402eaec4e525c40d
402eaea946da2be0
402eaf02bef6c319
402eaee70695fad1
402eaecb57060f67
402eaeafb1381ba0
402eaeeffd213733
402eaed45bb99444
402eaeb8c3c4ff03
402eae9d362d58db
402eaef66ca5c6cd
402eaedac5360ec6
402eaebf26f45366
402eaea392cca633
402eaefceb0a87c5
402eaee13d18aec8
402eaec598113662
402eaea9fce1c586
402eaf0378713787
402eaee7c38270dd
402eaecc173c1967
402eaeb0748f7326
402eaef0b8cdf0a2
402eaed51a91a2ca
402eaeb985a0a877
402eae9dfae55090
402eaef7356124dd
402eaedb9120a59b
402eaebff5e5e454
402eaea4649d08f5
402eaefdc09efed3
402eaee215e224af
402eaec673e6d879
402eaeaadb9b161a
402eaf045aa85f36
402eaee8a8f63fd8
402eaeccffc3391a
402eaeb16000baed
3fb68a0f0bd0e209
3fb68a0fbc56a981
3fb68a106e57c45a
3fb68a1121d0b61b
3fb68a0ee9ced02a
3fb68a0f9a5e3f13
3fb68a104c6ad830
3fb68a10fff10cbf
3fb68a0ec74434f9
3fb68a0f77dcabef
3fb68a1029f42be4
3fb68a10dd871311
3fb68a0ea433aef9
3fb68a0f54d49a35
3fb68a1006f67511
3fb68a10ba9589e7
3fb68a0f252bd097
3fb68a0fd607ca57
3fb68a10885dce95
3fb68a113c2a4f6f
3fb68a0f035fcc93
3fb68a0fb4475f5e
3fb68a1066aad09e
3fb68a111a867fac
3fb68a0ee109fd01
3fb68a0f91fc8bb4
3fb68a10446cd4e4
3fb68a10f857246d
3fb68a0ebe2cf1f2
3fb68a0f6f29eb34
3fb68a1021a682e5
3fb68a10d59ef093
