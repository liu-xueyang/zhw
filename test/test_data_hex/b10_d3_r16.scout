{
"dims": 2, 
"dataset": "S3D",
"rate": 16, 
"blocks": 10, 
"minbits": 256, 
"maxbits": 256, 
"maxprec": 64, 
"minexp": -1074
},
> 0x0bcfcab90afaaa970
> 0x0bce1e196dc623226
> 0x0bceec9b5bb4580ba
> 0x0bcfb27332fbc844b
> 0x0bcfcab8c1b3b30ef
> 0x0bcfcab896e34114f
> 0x0bcf5c0b13c61dde1
> 0x0bcfcab83f921f66d
> 0x0bcf65a6c057d8e23
> 0x0bcf95a773f301b6e
> 0x0bcf673e5473ad5f6
> 0x0bcfc62df6cae3d82
> 0x0bcfcab82ca487a02
> 0x0bcfaf845cf45804d
> 0x0bcf82b8940e1ae04
> 0x0bcf55e9db25d772c
> 0x03d2f61512ed4c7e1
> 0x03d14e812b8d08922
> 0x0bcfcab92701bbcfd
> 0x0bcfcab8fbef1ed57
> 0x0bcfcab9311e4a36e
> 0x0bcf628255b71d6c9
> 0x0bcfcab8dcb99671c
> 0x03cd33f5e529a607d
> 0x0bcf9aa63526604f1
> 0x0bcfc483f4178fa4e
> 0x0bcfcab89158e5d0d
> 0x0bcf387ff45c25535
> 0x0bcfcab2b1323f1d0
> 0x0bcfa70f55817d254
> 0x0bcf7c2c74cb181fe
> 0x0bcfcab81a5ece290
> 0x0bcfcab9ed0aba4f5
> 0x0bcfcab9c40410eea
> 0x03d1b0622c9493add
> 0x03d330a4540fc1b04
> 0x03d36d144b2db35f1
> 0x03d22ecfec82f9123
> 0x0bcf6de1244cb1909
> 0x0bcfcab92535172c3
> 0x0bcfcab955defa5fd
> 0x0bcfcab92ccd9e573
> 0x0bcf284a34378672b
> 0x0bcfcab8d8e1f0e16
> 0x0bced156e3cb32ae4
> 0x0bcf9171d13ae29d8
> 0x0bcfcab8b5d3cd9fe
> 0x0bcf5d097c85963ff
> 0x03d69cee43c6e5466
> 0x03d6229c5a69a6548
> 0x0bcfcaba0f9ede9a2
> 0x0bcfcab9e62f7b38f
> 0x0bcfcaba145d065c3
> 0x0bcfcab9ec18d3197
> 0x03d1642d560d4f3ab
> 0x03d210819f4b7f014
< 0x061157a231848b7a1
> 0x03d33535d4616ad9b
> 0x03d29d988712a4bbd
> 0x03d01360580e87bd5
> 0x0bcfcab94ce9a22f1
< 0x0516c247e01f1be77
> 0x0bcfcab97919896c0
> 0x0bcfcab950d907a26
> 0x0bcfcab9280b363c9
> 0x03cc470b3bc7366a3
< 0x0431980b86c9b4887
> 0x0402eaeef04cb746d
> 0x0402eaed35c5f04e7
> 0x0402eaeb7bdb2f4b1
> 0x0402eae9c29afd73f
< 0x0bafbcf76a88b9722
> 0x0402eaef55860160d
> 0x0402eaed9a9e88383
> 0x0402eaebe04ed1cd0
> 0x0402eaea26a59d8f2
> 0x0402eaefbbb3f60f0
> 0x0402eaee0063f070f
> 0x0402eaec45a7824ac
> 0x0402eaea8b8d84aec
> 0x0402eaf022d8d0ac6
> 0x0402eaee671855ee9
> 0x0402eaecabe7610a7
> 0x0402eaeaf15503427
< 0x05429e487ff1c87a7
> 0x0402eaeef6c05a4d5
> 0x0402eaed3c7387b0e
> 0x0402eaeb82c05726a
> 0x0402eae9c9b5598f6
< 0x0edee8b82b9152f43
> 0x0402eaef5cddf9374
> 0x0402eaeda2308065c
> 0x0402eaebe8185f7af
> 0x0402eaea2ea454fe6
< 0x0ca12e6ef78251009
> 0x0402eaefc3ecf0e8d
> 0x0402eaee08d713bd1
> 0x0402eaec4e525c40d
> 0x0402eaea946da2be0
< 0x060152358294ce29f
> 0x0402eaf02bef6c319
> 0x0402eaee70695fad1
> 0x0402eaecb57060f67
> 0x0402eaeafb1381ba0
> 0x0402eaeeffd213733
> 0x0402eaed45bb99444
> 0x0402eaeb8c3c4ff03
> 0x0402eae9d362d58db
> 0x0402eaef66ca5c6cd
> 0x0402eaedac5360ec6
> 0x0402eaebf26f45366
< 0x04086509c430c87a9
> 0x0402eaea392cca633
> 0x0402eaefceb0a87c5
> 0x0402eaee13d18aec8
> 0x0402eaec598113662
< 0x03fbe451fd4476a04
> 0x0402eaea9fce1c586
> 0x0402eaf0378713787
> 0x0402eaee7c38270dd
< 0x08f193633c44b1ef0
< 0x0273f5282e9ebc14c
> 0x0402eaecc173c1967
> 0x0402eaeb0748f7326
> 0x0402eaef0b8cdf0a2
> 0x0402eaed51a91a2ca
> 0x0402eaeb985a0a877
> 0x0402eae9dfae55090
> 0x0402eaef7356124dd
> 0x0402eaedb9120a59b
> 0x0402eaebff5e5e454
< 0x0beb7449b6ebb87af
> 0x0402eaea4649d08f5
> 0x0402eaefdc09efed3
> 0x0402eaee215e224af
> 0x0402eaec673e6d879
< 0x0d8bbaed38ed67674
> 0x0402eaeaadb9b161a
> 0x0402eaf045aa85f36
> 0x0402eaee8a8f63fd8
< 0x098d1022e41e53f74
< 0x09e0c03466027a41c
> 0x0402eaeccffc3391a
> 0x0402eaeb16000baed
> 0x03fb68a0f0bd0e209
> 0x03fb68a0fbc56a981
> 0x03fb68a106e57c45a
> 0x03fb68a1121d0b61b
> 0x03fb68a0ee9ced02a
> 0x03fb68a0f9a5e3f13
> 0x03fb68a104c6ad830
> 0x03fb68a10fff10cbf
> 0x03fb68a0ec74434f9
> 0x03fb68a0f77dcabef
> 0x03fb68a1029f42be4
> 0x03fb68a10dd871311
> 0x03fb68a0ea433aef9
> 0x03fb68a0f54d49a35
< 0x01235d10111006807
< 0x00a78a082e5cb833a
< 0x063987b0f074d5df5
< 0x019a9c94c37b1aafc
> 0x03fb68a1006f67511
> 0x03fb68a10ba9589e7
> 0x03fb68a0f252bd097
> 0x03fb68a0fd607ca57
> 0x03fb68a10885dce95
> 0x03fb68a113c2a4f6f
> 0x03fb68a0f035fcc93
> 0x03fb68a0fb4475f5e
> 0x03fb68a1066aad09e
> 0x03fb68a111a867fac
> 0x03fb68a0ee109fd01
> 0x03fb68a0f91fc8bb4
> 0x03fb68a10446cd4e4
> 0x03fb68a10f857246d
> 0x03fb68a0ebe2cf1f2
> 0x03fb68a0f6f29eb34
< 0x01235d10111006807
< 0x0887cbc92e5eb83ba
< 0x04ddc0b5d81ed5df5
< 0x0de1a327518f193d2
> 0x03fb68a1021a682e5
> 0x03fb68a10d59ef093
< 0x01235d10111006807
< 0x09f7ca8aa85e30bb2
< 0x04c1ccb4380ed99fc
< 0x0ea0b5275fcf3429c
< 0x09235d10111006807
< 0x09a78a89285c9833a
< 0x0555f3b4282cd4dee
< 0x0b7a165320ccd6002
< 0x0a6014044045567f9
< 0x020e12c1d5439013d
< 0x0b7f5bce0eb29a8a1
< 0x00641f454e5eb65d8
< 0x026014044045567f9
< 0x030b16d5d95b98139
< 0x0a4a5dae24bb9f0b1
< 0x001a1d584c18f2157
{
"Cycles/block": 0.000000, 
"Bytes/block": 128.000000, 
"FPGA clockrate": 5.000000e+07, 
"Throughput": inf
}
