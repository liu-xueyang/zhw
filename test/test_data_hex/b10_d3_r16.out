61157a231848b7a1
516c247e01f1be77
431980b86c9b4887
bafbcf76a88b9722
5429e487ff1c87a7
edee8b82b9152f43
ca12e6ef78251009
60152358294ce29f
4086509c430c87a9
3fbe451fd4476a04
8f193633c44b1ef0
273f5282e9ebc14c
beb7449b6ebb87af
d8bbaed38ed67674
98d1022e41e53f74
9e0c03466027a41c
1235d10111006807
0a78a082e5cb833a
63987b0f074d5df5
19a9c94c37b1aafc
1235d10111006807
887cbc92e5eb83ba
4ddc0b5d81ed5df5
de1a327518f193d2
1235d10111006807
9f7ca8aa85e30bb2
4c1ccb4380ed99fc
ea0b5275fcf3429c
9235d10111006807
9a78a89285c9833a
555f3b4282cd4dee
b7a165320ccd6002
a6014044045567f9
20e12c1d5439013d
b7f5bce0eb29a8a1
0641f454e5eb65d8
26014044045567f9
30b16d5d95b98139
a4a5dae24bb9f0b1
01a1d584c18f2157
